src/Sentry.Unity/Integrations/AnrIntegration.cs

```diff
@@ -8,76 +8,76 @@
 
 namespace Sentry.Unity
 {
-    internal class ANRIntegration : ISdkIntegration
+    internal class AnrIntegration : ISdkIntegration
     {
-        private static readonly object _lock = new object();
-        private static ANRWatchDog? _watchdog;
-        private SentryMonoBehaviour _monoBehaviour;
+        private static readonly object Lock = new();
+        private static AnrWatchDog? Watchdog;
```

Comment:
am I missing something? We're using `_` prefix for private members AFAICS

Reply:
Good catch! Yes, but not for `static`. I "fixed" it where I found it. (except for the SentryMonoBehaviour)